src/Filesystem.cpp

```diff
@@ -110,59 +103,30 @@ bool Filesystem::mount(const std::string& path) const
  */
 StringList Filesystem::searchPath() const
 {
-	if (!PHYSFS_isInit()) { throw filesystem_not_initialized(); }
-
-	StringList searchPath;
-
-	auto searchPathList = PHYSFS_getSearchPath();
-	for (char **i = searchPathList; *i != nullptr; ++i)
-	{
-		searchPath.push_back(*i);
-	}
-	PHYSFS_freeList(searchPathList);
+    if(!isInit()) { throw filesystem_not_initialized(); }
 
+    StringList searchPath{};
+    for(const auto& p : this->mSearchPath) {
```

Comment:
Brace placement. Condense into a single line:

'for(const auto& p : this->mSearchPath) { searchPath.push_back(p.string()); }